packages/compiler/src/template/pipeline/src/phases/reify.ts

```diff
@@ -698,6 +687,46 @@ function reifyUpdateOperations(unit: CompilationUnit, ops: ir.OpList<ir.UpdateOp
   }
 }
 
+/**
+ * Converts an ARIA property name to its corresponding attribute name, if necessary.
+ *
+ * For example, converts `ariaLabel` to `aria-label`.
+ *
+ * https://www.w3.org/TR/wai-aria-1.2/#accessibilityroleandproperties-correspondence
+ */
+function ariaAttrName(name: string): string {
+  return name.charAt(ARIA_PREFIX.length) !== '-'
+    ? ARIA_PREFIX + '-' + name.slice(ARIA_PREFIX.length).toLowerCase()
+    : name; // Property already has attribute name.
```

Comment:
```suggestion
  return name.startsWith('aria-') ? 
      name : `aria-${name.slice(4).toLowerCase()}`;
```
Is there any reason you couldn't just use `startsWith` here?

Reply:
A (undocumented) precondition of this function is that we _know_ the name begins with `aria`, because that's the circumstance that leads to performing this mapping.

I'll document this precondition, but if you strongly prefer the `startsWith` check I can do that instead.